src/jdk.accessibility/windows/native/common/AccessBridgeDebug.cpp

```diff
@@ -144,37 +144,10 @@ char *printError(char *msg) {
         }
     }
 
-    /**
-     * Send Java debugging info to the appropriate place
-     */
```

Comment:
Deleting this macro - without comment !  - seems to be somewhat out of scope here.

Reply:
Reverted.

I removed this function because its definition (`PrintJavaDebugString2`) did not match the declaration (`PrintJavaDebugString`), so the function wasn't really usable. I'm fine with leaving it as is, or cleaning up separately.